src/drivers/WinAPI/Fl_WinAPI_Window_Driver.cxx

```diff
@@ -475,8 +475,13 @@ void Fl_WinAPI_Window_Driver::hide() {
     }
   }
 
-  if (hide_common()) return;
-
+  if (hide_common()) {
+    if (count) {
```

Comment:
Using `if (count)` is **_potentially_** not correct here because `count` is reset to `0` in line 470 and not necessarily incremented in line 473 (this would depend on another condition).
The correct condition here would be `if (doit)` or just use `delete[] doit;` unconditionally because the argument of `delete[]` is allowed to be a NULL pointer.

Reply:
The same (count may be 0) applies to line 522 (in this modified source file) which should be fixed independently (I will do this in another commit).

Note to myself: the statement `delete[] doit;` in line 522 should be moved below the closing `'}'`, i.e. outside the `if (count) {...}` condition.

Reply:
Fixed in commit https://github.com/fltk/fltk/commit/d3e8648402a5c1094843e17f8fe72fe628af6e69 (force-pushed).